test/test_cleanup.py

```diff
@@ -56,10 +57,12 @@ def test_cleanup_snapshots(tmp_path, monkeypatch):
     assert not p2.is_dir(), "snapshot directory is gone"
 
 
-def test_cleanup_author_files(tmp_path):
+def test_cleanup_author_files(tmp_path, caplog):
+    caplog.set_level(logging.INFO)
+
     # set up some timestamps
     now = datetime.now()
-    time_format = "%Y-%m-%d"
+    time_format = "%m-%d-%Y"
```

Comment:
fix the test to match the actual format of these files: 05-25-2025 and not 2025-05-25